### ISSUE TITLE
Which Layers To Freeze When Continuing With Different Dataset ?

### ISSUE THREAD

**Issue by cmdrootaccess** [NathanUA/U-2-Net#73]
Hello,

Which layers can we freeze if we want to train the saved model with a different dataset ?

**Comment by SreenijaK** [comment]
@cmdrootaccess  did u get the answer .? which layer did u freeze?

**Comment by cmdrootaccess** [comment]
@SreenijaK no, but resuming training with old and new dataset works. thats how i have done it, in the meantime
